.gitlab/ci/.gitlab-ci.slack-notify.yml

```diff
@@ -1,11 +1,11 @@
 
 default:
-  image: docker-io.art.code.pan.run/devdemisto/gitlab-content-ci:1.0.0.49555
```

Comment:
There are 2 docker registries?

1. docker.art.code.pan.run
2. docker-io.art.code.pan.run

Reply:
Yes, `docker-io.art.code.pan.run` is a caching proxy for `docker.io` the other one is the a regular repo